k8s.io/ingress-prod.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: k8s-io
+  labels:
+    app: k8s-io
+  namespace: k8s-io-prod
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: k8s-io-ingress-prod
+spec:
+  tls:
+  - secretName: ssl
```

Comment:
I assume the cert manager work comes after the flip?

Reply:
right, because I can't do the http challenges until the domain is pointing at the Ingress.

... which makes me realize that I'm not sure if we'll be able to generate a cert for the canary instance, since the domains don't point at it. hm.

Reply:
I guess I could create a [self-signed `ClusterIssuer`](https://cert-manager.readthedocs.io/en/latest/tasks/issuers/setup-selfsigned.html), since we don't need real certs for the canary instance.